go/runtime/bundle/component.go

```diff
@@ -255,6 +260,35 @@ func (r *TDXResources) Validate() error {
 	if r.CPUCount < 1 {
 		return fmt.Errorf("vCPU count must be at least 1")
 	}
+	if r.GPU != nil {
+		err := r.GPU.Validate()
+		if err != nil {
```

Comment:
```suggestion
		if err := r.GPU.Validate(); err != nil {
```

Reply:
I believe this will cause a lint issue due to nested ifs.